.config/nextest.toml

```diff
@@ -9,12 +9,21 @@ failure-output = "immediate-final"
 fail-fast = false
 
 [test-groups]
-serial = { max-threads = 1 }
+serial-cli = { max-threads = 1 }
+serial-metrics = { max-threads = 1}
 
 [[profile.default.overrides]]
-filter = 'test(/_serial$/)'
-test-group = 'serial'
+filter = 'test(/_cli_serial$/)'
+test-group = 'serial-cli'
+
+[[profile.default.overrides]]
+filter = 'test(/_metrics_serial$/)'
+test-group = 'serial-metrics'
+
+[[profile.ci.overrides]]
+filter = 'test(/_cli_serial$/)'
```

Comment:
double check: do we def need the two groups here. I'm thinking serial may be enough (after the changes we discussed)?

Reply:
cc @bgins